src/Compilation.zig

```diff
@@ -5344,8 +5344,10 @@ pub fn addCCArgs(
                 "-nostdinc",
                 "-fno-spell-checking",
             });
-            if (comp.config.lto) {
-                try argv.append("-flto");
+            switch (comp.config.lto) {
+                .none => {},
```

Comment:
```suggestion
                .none => try argv.append("-fno-lto"),
```

Missed this earlier; we don't want to rely on Clang defaults if we can help it since they can change over time.